src/main/java/io/apicurio/datamodels/asyncapi/models/AaiComponents.java

```diff
@@ -0,0 +1,39 @@
+package io.apicurio.datamodels.asyncapi.models;
+
+import io.apicurio.datamodels.core.models.ExtensibleNode;
+
+/**
+ * @author Jakub Senko <jsenko@redhat.com>
+ */
+public abstract class AaiComponents extends ExtensibleNode {
+
+    /**
+     * NOT Required.
+     */
+    public AaiSchemaItems schemas;
```

Comment:
I think most of these should just be `Map<String, Aai*>`.  See `Oas30Components` as an example.